deploy-prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+cd /var/www/dds
+
+# Update mes-aides
+git fetch origin
+git rebase origin/master
+npm install
+grunt build
+
+# Update openfisca
+npm run install-openfisca
+cat openfisca/api_config.ini | sed s/"port = 2000"/"port = 12000"/ > ~/production.ini
```

Comment:
The port and destination file should be `$VARIABLES` defined at the top of the script, to increase the discoverability of the configuration points.